src/main/java/usdot/fhwa/stol/c2c/c2c_mvt/parsers/NGTMDDJsonParser.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2025 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package usdot.fhwa.stol.c2c.c2c_mvt.parsers;
+
+import usdot.fhwa.stol.c2c.c2c_mvt.C2CMVTException;
+import usdot.fhwa.stol.c2c.c2c_mvt.messages.JsonC2CMessage;
+
+
+/**
+ * Implementation of {@link Parser} for NGTMDD Messages
+ * 
+ * @author Aaron Cherney
+ */
+public class NGTMDDJsonParser extends JsonParser
```

Comment:
What is the difference between  NGTMDD message and C2Cmessage? Do we need to support JSON along with other message format, like XML? Why cannot the NGTMDDParser does both XML and JSON parsing rather than introducing an intermediary jsonParser class?

Reply:
C2CMessage (now C2CBaseMessage) is the base class for any message of any C2C Standard. ngTMDD is a specific C2C Standard so it has an implementation which is JSON since the ngTMDD data format is JSON. If we implement a C2C Standard that has XML as the data format we will need to implement a XMLC2CMessage/Decoder/Parser.